utils/utils.py

```
@@ -23,9 +23,11 @@
 
 def flatten_dict_observations(env):
     assert isinstance(env.observation_space, gym.spaces.Dict)
-    keys = env.observation_space.spaces.keys()
-    return gym.wrappers.FlattenDictWrapper(env, dict_keys=list(keys))
-
+    try:
+        return gym.wrappers.FlattenObservation(env)
+    except:
```

please do not use bare except

---

Will fix, thanks for the feedback